src/Cake.Prca.Tests/FakeCodeAnalysisProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cake.Prca.Tests
+{
+    using System.Collections.Generic;
+    using Core.Diagnostics;
+    using Prca.Issues;
+
+    public class FakeCodeAnalysisProvider : CodeAnalysisProvider
+    {
+        private readonly List<ICodeAnalysisIssue> issues = new List<ICodeAnalysisIssue>();
+
+        public FakeCodeAnalysisProvider(ICakeLog log) 
+            : base(log)
+        {
+        }
+
+        public FakeCodeAnalysisProvider(ICakeLog log, IEnumerable<ICodeAnalysisIssue> issues)
+            : base (log)
```

Comment:
I don't meant to use the very exact same ruleset. We never do that, but I meant not using analyzers at all would feel wrong for me.